src/SFA.DAS.AssessorService.Data/CertificateRepository.cs

```diff
@@ -53,11 +93,13 @@ public async Task<Certificate> New(Certificate certificate)
                 if (sqlException.Number == 2601 || sqlException.Number == 2627)
                 {
```

Comment:
My previous comment about exception as flow is true of this too. Again, I vote for don't hold up this story, put this and the other one it a little spike investigation for a future date.